circle.yml

```diff
@@ -39,12 +36,11 @@ dependencies:
     - ./bin/circleci/build-version-json.sh
     - ./bin/circleci/build-addon.sh
     - ./bin/circleci/build-frontend.sh
-    - '[[ " $TESTPILOT_STATIC_BRANCHES " =~ " $CIRCLE_BRANCH " ]] || ./bin/circleci/build-server.sh'
 
 test:
   override:
-    - '[[ " $TESTPILOT_STATIC_BRANCHES " =~ " $CIRCLE_BRANCH " ]] || ./bin/circleci/run-server-unit-tests.sh'
-    # - ./bin/circleci/run-integration-tests.sh
+    - echo "Skipping integration tests."
+    # - ./bin/circleci/run-integration-tests.shi
 
 # appropriately tag and push the container to dockerhub
 deployment:
```

Comment:
Docker-dependent stuff below here that will break on merge to master. Should delete the `hub_*` deployment cases, leaving the `static_deployment` bit. 

I'd also change the branch from `development-static` to `master`, which will deploy to testpilot-static.dev.mozaws.net. But, I can follow up on that in #1522